ballerina/tests/fromXml_test.bal

```diff
@@ -2812,6 +2812,36 @@ function testProjectionWithXmlAttributeForParseAsType() returns error? {
     test:assertEquals(rec.A, "2");
 }
 
+type RecType record {
+    string name;
+    @Name {
+        value: "name"
+    }
+    @Attribute
+    string duplicateName;
```

Comment:
Can you add a test where this duplicate field being record.

Reply:
Added https://github.com/ballerina-platform/module-ballerina-data.xmldata/pull/29/commits/efe4da67d8f15b671d88b7b9179f7793c15280fb